INSTALL

```diff
@@ -133,16 +134,33 @@ Compiling for BlueGene/Q
 NEST provides a cmake tool-chain file for cross compilation for BlueGene/Q. When
 configuring NEST use the following `cmake` line:
 
-    cmake -DCMAKE_TOOLCHAIN_FILE=Platform/BlueGeneQ_XLC 
-          -DCMAKE_INSTALL_PREFIX:PATH=</install/path>
-          -Dwith-python=OFF
-          -Dstatic-libraries=ON
+    cmake -DCMAKE_TOOLCHAIN_FILE=Platform/BlueGeneQ_XLC \
+          -DCMAKE_INSTALL_PREFIX:PATH=</install/path> \
+          -Dwith-python=OFF \
+          -Dstatic-libraries=ON \
           </path/to/NEST/src>
 
-It is recommended to build statically on larger BlueGene/Q systems. If you need
-PyNEST on BlueGene/Q, you have to compile dynamically, i.e. `-Dstatic-libraries=OFF`
-and you have to cythonize the `pynest/pynestkernel.pyx/.pxd`
-on a machine with Cython installed:
+If you compile dynamically, be aware that the BlueGene/Q system might not provide a `ltdl`
+library. If you want to dynamically load an external user module, you have to
+compile and install a `ltdl` yourself and add `-Dwith-ltdl=<ltdl-install-dir>`
+to the `cmake` line. Otherwise add `-Dwith-ltdl=OFF`.
+
+Additionally, the design of cmake's mpi handling has a broken design, which is
+brittle in the case of BGQ and certain libraries (flags to use sionlib, for example).
+
+If you run into that, you must force cmake to use the wrappers rather than
+it's attempts to extract the proper flags for the underlying compiler
+as in:
+
+    -DCMAKE_C_COMPILER=/bgsys/drivers/ppcfloor/comm/xl/bin/mpixlc_r
+    -DCMAKE_CXX_COMPILER=/bgsys/drivers/ppcfloor/comm/xl/bin/mpixlcxx_r
+
+BlueGene/Q and PyNEST
+---------------------
+
+Building PyNEST on BlueGene/Q requires you to compile dynamically, i.e. 
+`-Dstatic-libraries=OFF`. Further, you have to cythonize the
+`pynest/pynestkernel.pyx/.pxd` on a machine with Cython installed:
```

Comment:
@heplesser `.pxd` is the equivalent of a cython header, and goes with the `.pyx`. Will change that to pyx, for simplicities sake.